src/main/java/com/jsyn/apps/InstrumentTester.java

```diff
@@ -89,7 +89,7 @@ public void close() {
         @Override
         public void send(MidiMessage message, long timeStamp) {
             byte[] bytes = message.getMessage();
-            messageParser.parse(bytes);
+            messageParser.parse(bytes, /* offset= */ 0, bytes.length);
```

Comment:
parse(bytes) is a public method and should be preserved.
Otherwise this change could break existing code.

Add a method:
    public void parse(byte[] message) {
        parse(message, 0, message.length);
    }
    
in MessageParser.java.
Then you can remove the change in this file.

Reply:
Makes sense. Done.